mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraAttachmentMapper.java

```diff
@@ -52,17 +57,21 @@
 import com.github.fge.lambdas.ThrownByLambdaException;
 import com.github.steveash.guavate.Guavate;
 import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableSet;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class CassandraAttachmentMapper implements AttachmentMapper {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CassandraAttachmentMapper.class);
     private final CassandraAsyncExecutor cassandraAsyncExecutor;
+    private MessageId.Factory messageIdFactory;
 
     @Inject
     public CassandraAttachmentMapper(Session session) {
         this.cassandraAsyncExecutor = new CassandraAsyncExecutor(session);
+        this.messageIdFactory = new CassandraMessageId.Factory();
```

Comment:
The messageIdFactory should be injected, like anywhere else.

Reply:
ok